crates/query-engine/execution/src/error.rs

```diff
@@ -4,7 +4,7 @@ pub enum Error {
     #[error("{0}")]
     Query(QueryError),
     #[error("{0}")]
-    DB(sqlx::Error),
```

Comment:
Very surprised to see we must still have a dependency on `sqlx`

Reply:
Yeah something around metrics seems to be around. Probably should fix/clean that, but for another PR.